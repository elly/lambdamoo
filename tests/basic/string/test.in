length("foo")
strsub("foobarbaz", "bar", "quxx")
index("foobar", "bar")
index("foobar", "baz")
index("fooBar", "bar")
index("fooBar", "baz")
index("fooBar", "bar", 1)
rindex("bazbarbazfoo", "baz")
rindex("bazbarbazfoo", "quxx")
rindex("bazbarBazfoo", "baz")
rindex("bazbarBazfoo", "baz", 1)
strcmp("abc", "abd") < 0
strcmp("abd", "abc") > 0
strcmp("abc", "abc")
decode_binary("foo")
decode_binary("foo", 1)
encode_binary({102, 111, 111})
encode_binary({"fo", 111}, "f", {111, 111})
crypt("foobar", "SA")
string_hash("foobar")
