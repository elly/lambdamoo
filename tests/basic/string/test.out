3
"fooquxxbaz"
4
0
4
0
0
7
0
7
1
1
1
0
{"foo"}
{102, 111, 111}
"foo"
"foofoo"
"SAEmC5UwrAl2A"
"3858F62230AC3C915F300C664312C63F"
